reporter/src/funTest/kotlin/reporters/CycloneDxReporterFunTest.kt

```diff
@@ -38,26 +37,27 @@ import org.ossreviewtoolkit.reporter.ReporterInput
 import org.ossreviewtoolkit.utils.test.createSpecTempDir
 
 class CycloneDxReporterFunTest : WordSpec({
+    val defaultSchemaVersion = CycloneDxReporter.DEFAULT_SCHEMA_VERSION.versionString
```

Comment:
commit message: Please provide a rationale why it is needed to use multiple schema versions.